CONTRIBUTING.md

```diff
@@ -57,6 +57,8 @@ Setup environment:
   - For ubuntu and debian run `sudo apt-get install build-essentials`
 - Verify that everything is installed by running `make build`
 
+Note: In `make build`, the verbosity (`-v` flag) is enabled by default. Any additional go build flags, if required, should be added to env `GO_BUILD_FLAGS` before running `make build`.
```

Comment:
Don't think this information is relevant enough to be put in contributing guide. I would prefer to keep them concise so new contributors will not be overwhelmed with information.

Reply:
Makes sense. I'll try simplifying it.

Reply:
Done. Please check now @serathius